### PR TITLE
Add and Remove books from the list

### PR DESCRIPTION
**Pull request by Azonkeu**
Hi 👋!

- [ ] This pull request is to add new features to the webpage. Users can add new books to the list by filling out the form. They can also remove books from the list by clicking on the "remove" button.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    window.location.reload();
+  } else {
+    bookArr.push(book);
+    const converted = JSON.stringify(bookArr);
+    localStorage.setItem('data', converted);
+    window.location.reload();
+  }
+};
+
+// eslint-disable-next-line no-unused-vars
```
Comment (mikenath223) [inline script.js]: - [ ] Kindly remove this eslint disable line its best to fix the issue in the code.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+function removeBook(id) {
+  const data = localStorage.getItem('data');
+  const convertedBooks = JSON.parse(data);
+  const remainingBooks = convertedBooks.filter((book) => book.id !== id);
+  const removedBooks = JSON.stringify(remainingBooks);
+  localStorage.setItem('data', removedBooks);
+  window.location.reload();
+}
```
Comment (mikenath223) [inline script.js]: - [ ] Please change this to es6 arrow function syntax.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**